### PR TITLE
Add Docerina Links in Java Interop, IO, Common Libs, & Security BBEs

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to add Docerina Links in Java Interop, IO, Common Libs, & Security BBEs

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/java-arrays/java_arrays.description [hunk 1]
```diff
@@ -1,2 +1,4 @@
 // Java arrays are represented by the `handle` type in Ballerina. They can be manipulated using the
-// `getArrayElement`, `setArrayElement` and `getArrayLength` methods provided by the `ballerina/java` module.
+// `get`, `set` and `getLength` methods provided by the `ballerina/java.arrays` module.<br/><br/>
```
Comment (MaryamZi) [inline examples/java-arrays/java_arrays.description:2]: Shouldn't this be 
```suggestion
// `get`, `set`, and `getLength` methods provided by the `ballerina/java.arrays` module.<br/><br/>
```

File: examples/java-arrays/java_arrays.description [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+// For more information on the underlying modules, see
```
Comment (MaryamZi) [inline examples/java-arrays/java_arrays.description:3]:
```suggestion
// For more information on the underlying module, see
```

File: examples/record-io/record_io.description [hunk 3]
```diff
@@ -1 +1,3 @@
-//This example demonstrates how to read and write records using I/O APIs. 
+//This example demonstrates how to read and write records using I/O APIs.<br/><br/>
```
Comment (MaryamZi) [inline examples/record-io/record_io.description:1]:
```suggestion
// This example demonstrates how to read and write records using I/O APIs.<br/><br/>
```

File: examples/java-arrays/java_arrays.description [hunk 4]
```diff
@@ -1,2 +1,4 @@
 // Java arrays are represented by the `handle` type in Ballerina. They can be manipulated using the
-// `getArrayElement`, `setArrayElement` and `getArrayLength` methods provided by the `ballerina/java` module.
+// `get`, `set`, and `getLength` methods provided by the `ballerina/java.arrays` module.<br/><br/>
+// For more information on the underlying module, see
+// [the java.arrays module](https://ballerina.io/learn/api-docs/ballerina/java.arrays/index.html)
```
Comment (daneshk) [inline examples/java-arrays/java_arrays.description:4]:
```suggestion
// [the Java Arrays module](https://ballerina.io/learn/api-docs/ballerina/java.arrays/index.html)
```

Comment (daneshk) [inline examples/java-arrays/java_arrays.description:4]:
```suggestion
// For more information on the underlying module, 
// see [the Java Arrays module](https://ballerina.io/learn/api-docs/ballerina/java.arrays/index.html)
```

Comment (daneshk) [inline examples/java-arrays/java_arrays.description:4]:
```suggestion
// For more information on the underlying module,
// see [the java.arrays module](https://ballerina.io/learn/api-docs/ballerina/java.arrays/index.html)
```

**Pull request merged**